src/IO/ITKImageInputFileFormat.cxx

```diff
@@ -440,11 +440,11 @@ read_file_itk(const std::string &filename)
           typedef itk::GDCMSeriesFileNames NamesGeneratorType;
           typename NamesGeneratorType::Pointer nameGenerator = NamesGeneratorType::New();
           nameGenerator->SetUseSeriesDetails( true );
-          // Make sure we read only data from a single frame and gate
+          // Reads complete series.
           nameGenerator->AddSeriesRestriction("0008|0022" ); // AcquisitionDate
```

Comment:
If all the others are removed, is this really necessary? The series number should be adequate no?

Reply:
Yes and no. Really it could be removed though my justifications for leaving it were:

- It does no harm.
- Say a participant had two studies on different days, the series number could be identical if the protocol was the same. If for some reason these were both placed in the same folder then there could be the risk of treating them as one. In reality this shouldn't happen, the studyUID *should* prevent it. However, if the UIDs have been re-written, for example during anonymisation, then it is plausible. 

Happy to remove though...

Reply:
definitely don't want to remove that one. It's easy to make different series with the same series number... (even within the same study)